README.md

```diff
@@ -30,10 +30,12 @@ We're happy to review and approve new filters and actions that help you integrat
 * Change the least amount of code to achieve the goal; PRs with lots of whitespace changes and abstraction combined with a feature add or bug fix are difficult to review and may be rejected
 * Use the latest version of WordPress and turn `WP_DEBUG` on
 * If other plugins are installed during testing that might affect behavior, please list those in the PR description
-* Run the [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) code quality scripts on the files changed:
-    * `composer compat` to check for PHP version compatibility (currently 5.3), ignore files that you have not modified
-    * `composer phpcbf-path ./path/to/file/changed.php` to clean up whitespace
-    * `composer phpcs-path ./path/to/file/changed.php` to find other potential issues
+* Run the [Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) scripts listed below before creating your PR:
+	* `composer phpcbf` to correct whitespace project-wide (required by CI)
+	* `composer compat` to check for PHP version compatibility (currently 5.3; required by CI)
+	* `composer phpcs-tests` to run code standard checks on test files (required by CI)
+	* `composer phpcs-path ./path/to/file/changed.php` to find other potential issues (not required by CI currently)
+	* `composer test` to run all tests (required by CI; see "Testing" section below)
```

Comment:
you could link to this very same file's section just as normal MD. Use the title as anchor.  see https://gist.github.com/asabaylus/3071099